stixcore/products/product.py

```diff
@@ -580,21 +580,20 @@ def __add__(self, other):
         other_data['old_index'] = [f"o{i}" for i in other_data['control_index']]
         self_data['old_index'] = [f"s{i}" for i in self_data['control_index']]
 
-        control = vstack((self_control, other_control))
+        if (self.service_type, self.service_subtype) == (3, 25):
```

Comment:
I'm not sure if we should do the same for other TM types basically anything not (21, 6)?

Reply:
lets see in future